### PR TITLE
Change profile picture

### PR DESCRIPTION
**Pull request by jamieliu386**

### CODE REVIEW

File: src/reducers/userDataReducers.js [hunk 1]
```diff
@@ -30,17 +32,19 @@ function userDataReducers(state = initialState, action) {
...
-      //   .catch(err => {
-      //     state.error = err
-      //     console.log(err)
-      //   })
-      return state;
+      let newName = action.value;
+      fetch
+        .updateUserData(state.uid, { displayName: newName })
+        .then(response => {
+          console.log(action.value);
```
Comment (connorbo97) [inline src/reducers/userDataReducers.js]: remove logs

File: src/reducers/userDataReducers.js [hunk 2]
```diff
@@ -30,17 +32,19 @@ function userDataReducers(state = initialState, action) {
...
+          //if nothing went bad, keep the display name, otherwise, change it back (or dont, depends how we wanna do it)
+          console.log(response);
+        })
+        .catch(err => {
+          state.error = err;
```
Comment (connorbo97) [inline src/reducers/userDataReducers.js]: can keep error log

File: src/styles/Panel.css [hunk 3]
```diff
@@ -38,16 +38,136 @@
 .panel-image {
   border-radius: 500px;
   border: 1px solid #dddcdf;
-  width: 205px;
+  width: 200px;
```
Comment (connorbo97) [inline src/styles/Panel.css]: i liked the slightly oblong image, but if you feel the circular looks better, I don't mind

Comment (jamieliu386) [reply src/styles/Panel.css]: ah ok i switched it back to 205px

File: src/components/Editor/components/ProfilePanel.js [hunk 4]
```diff
@@ -82,12 +267,11 @@ class ProfilePanel extends React.Component {
         {/*character is leftward facing arrow*/}
       </div>
       <div className="panel-content">
-        <img
-          className="panel-image"
-          src={this.props.photoURL ? this.props.photoURL : defaultPic}
-          alt="Your profile"
-        />{" "}
-        <div className="panel-name">{this.props.displayName || "Joe Bruin"}</div>{" "}
+        <div>{this.renderPanelImage()}</div>
```
[the remark this answers is not in the document]
Comment (jamieliu386) [reply src/components/Editor/components/ProfilePanel.js]: oh nope

**Pull request merged**